bool status_request
---
string next_action
string item